browser/src/control/Control.Command.js

```diff
@@ -81,19 +75,23 @@ L.Map.include({
 		});
 		message.push('</ul>', '</div>', '<div>');
 
-		vex.dialog.confirm({
-			unsafeMessage: message.join(''),
-			showCloseButton: false,
-			contentClassName: 'vex-content vex-locking ' + lockingOnMobile,
-			callback: function (value) {
-				if (value)
-					window.open(that.Locking.unlockLink, '_blank');
-			},
-			buttons: [
-				$.extend({}, vex.dialog.buttons.NO, { text: _('Cancel') }),
-				$.extend({}, vex.dialog.buttons.YES, { text: _('Unlock') })
-			]
-		});
+		message = message.join();
+
+		this.uiManager.showInfoModal('unlock-features-popup', this.Locking.unlockTitle, ' ', ' ', _('Unblock'), function() {
+			window.open(this.Locking.unlockLink, '_blank');
+			if (this.jsdialog)
+				this.jsdialog.closeAll();
+			else
+				this.fire('closemobilewizard');
+		}.bind(this), true);
+		document.getElementById('unlock-features-popup').querySelectorAll('p')[0].outerHTML = message;
+
+		document.getElementById('unlock-features-popup-cancel').onclick = function() {
+			if (this.jsdialog)
+				this.jsdialog.closeAll();
+			else
```

Comment:
instead of this conditional code you can simply use `closeModal` method: https://github.com/CollaboraOnline/online/blob/master/browser/src/control/Control.UIManager.js#L822

It sends jsdialog message so on mobile it is handled by mobile-wizard, on desktop by jsdialog control.